web/src/app/converters/submission-data-converter.ts

```diff
@@ -65,27 +61,28 @@ function taskDataPbToModel(pb: Pb.ITaskData[], job: Job): SubmissionData {
       takePhotoResult,
     } = taskData;
 
+    let value = null;
```

Comment:
@rfontanarosa  Can you extra this logic to its own method in follow-up PR (eg `taskValueToResult`)? Rather than relying on a mutable variable you could just `return` the desired value or `null` depending on